src/Castle.Windsor.Extensions.MsDependencyInjection/LoggerDependencyResolver.cs

```diff
@@ -0,0 +1,42 @@
+// Copyright 2004-2020 Castle Project - http://www.castleproject.org/
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+namespace Castle.Windsor.Extensions.MsDependencyInjection
+{
+    using Castle.Core;
+    using Castle.MicroKernel;
+    using Castle.MicroKernel.Context;
+    
+    using Microsoft.Extensions.Logging;
```

Comment:
Comes with Microsoft.Extensions.Hosting but added as explicit as well. What's the policy on transient dependencies?

Reply:
> What's the policy on transient dependencies?

We don't really have a policy, but I know Microsoft recently removed some transient dependencies between ASP.NET packages that weren't needed, which breaks situations like this. I think if the Windsor code uses the public API of the Logging package it should define an explicit dependency rather than just relying on it being there because something else brings it in.